int64 id
int64 age
string test_type
string informant
string informant_accuracy
float64 p_chooses_with_informant



